packages/cli/src/public-api/v1/handlers/users/users.handler.ee.ts

```diff
@@ -82,8 +82,16 @@ export = {
 	createUser: [
 		globalScope('user:create'),
 		async (req: Create, res: Response) => {
-			const usersInvited = await Container.get(InvitationController).inviteUser(req);
+			const { data, error } = InviteUsersRequestDto.safeParse(req.body);
+			if (error) {
+				return res.status(400).json(error.errors[0]);
```

Comment:
does the error object has the same structure as in the documentation? I think the ZodError has a different structure

![CleanShot 2024-12-26 at 11 49 14@2x](https://github.com/user-attachments/assets/11e5a8a3-2ab6-4ccc-a8f2-f198d6b39c42)


![CleanShot 2024-12-26 at 11 47 13@2x](https://github.com/user-attachments/assets/5841294e-24a5-48f9-876f-66f9f313487e)

Reply:
I guess the API should have a different error structure, to be able to return an array of errors instead of a single error object

Reply:
I'll have a look at updating all validation failure responses to have a consistent structure in a separate PR.